docs/resources/cassandra_keyspace.md

```diff
@@ -10,7 +9,40 @@ description: |-
 
 Resource schema for AWS::Cassandra::Keyspace
 
+## Example Usage
 
+To create a SQS queue with tags
```

Comment:
this title does not match the actual template file, I suspect you need to re-run `make docs`

Reply:
Yeah, its because I missed that in the initial commit. There is another commit in this PR which has it fixed. This is the old commit. Correct Commit id is : e9f43ede8f0791acedfc6079031c9351a94d879d

Reply:
yup, however you still need to re-run `make docs` locally and that should re-create the `cassandra_keyspace.md` with the changes

Reply:
ah I just missed that. Thanks for catching it. Changes Done.